catroid/src/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -354,6 +354,10 @@ private Object interpretFunction(Functions function, Sprite sprite) {
 				return doubleValueOfLeftChild == null ? 0d : java.lang.Math.toDegrees(Math.atan(doubleValueOfLeftChild));
 			case EXP:
 				return doubleValueOfLeftChild == null ? 0d : java.lang.Math.exp(doubleValueOfLeftChild);
+			case POWER:
+				return ((doubleValueOfLeftChild == null || doubleValueOfRightChild == null)
+						|| (doubleValueOfLeftChild < 0 && doubleValueOfRightChild < 1 && doubleValueOfRightChild > -1))
```

Comment:
Many apologies! There has been a twist in our thinking process :scream_cat: Of course 
```Java
Math.pow(-1,0)
```

should be 1! Mhm since there are a truckload of special cases wouldnt it be better to use try and catch blocks? I know they are a bit "ugly" because they disrupt the current layout of the cases. On the other hand it would cover all special cases as listet in the [documentation](https://developer.android.com/reference/java/lang/Math.html#pow(double,%20double)). Ill try some tests but I have to restart my PC for it so this is it for now :smiley_cat: